assets/templates/kubernetes.node/configs.yml

```diff
@@ -0,0 +1,33 @@
+fields:
+  - name: cloud.availabilit_zone
+    value: "europe-west1-d"  
+  - name: index
+    value: "metrics-kubernetes.node-default"  
+  - name: agent.id
+    enum: ["i-1", "i-2", "i-3", "i-4","i-5", "i-6", "i-7", "i-8","i-9", "i-10", "i-11", "i-12","i-13", "i-14", "i-15", "i-16"]
+  - name: agent.name
+    value: "kubernetes-scale-123456"
+  - name: agent.ephemeral_id  
+    value: "f94220b0-2ca6-4809-8656-eb478a66c541"
+  - name: agent.version
+    value: "8.7.0"
+  - name: agent.snasphost
+    value: false
+  - name: metricset.period
+    value: 10000
+  - name: event.duration
+    range:
+      min: 1
+      max: 4000000
+  - name: rangeofid
+    range:
+      min: 0
+      max: 10000
+  - name: timedate
```

Comment:
you can have a date type with `period` ([values will be evenly generated between `time.Now()` and `time.Now().Add(period)`, where `period` is expressed as `time.Duration`](https://github.com/elastic/elastic-integration-corpus-generator-tool/blob/main/docs/fields-configuration.md))

along with `-n, --now date               time to use for generation based on now (date type)` when invoking `elastic-integration-corpus-generator-tool generate-with-template`

Reply:
Do you have any example on how to make this work? I did not find any other usage of `period`. If I set the field as `10s` how can I then use it in the tpl file?

Reply:
Interesting!

Reply:
@aspacca I tried this out and the timestamps created are in the format `2023-10-06 11:42:06.999999 +0000 UTC` which is not [strict_date_optional_time||epoch_millis](https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-date-format.html). They seem to cannot be parsed by ES when trying to upload the generated events.

Reply:
Done in https://github.com/elastic/elastic-integration-corpus-generator-tool/pull/107/commits/6991fc4d50efa475b37c4eec87899986f770efe4